mibs/hillstone/HILLSTONE_TEMPERATURE_MIB.my

```diff
@@ -0,0 +1,94 @@
+-- **************************************************************************

```

Comment:
Please drop the `.my` from the filename and rename it `HILLSTONE-TEMPERATURE-MIB`